src/coreclr/jit/lsraarmarch.cpp

```diff
@@ -331,6 +331,24 @@ int LinearScan::BuildCall(GenTreeCall* call)
         }
     }
 
+    if (call->gtCallType == CT_INDIRECT)
```

Comment:
Can you add a comment here? Why do we need this case for just TLS and not for any previous call case?